src/httpfetch.cpp

```diff
@@ -386,8 +387,11 @@ const HTTPFetchResult * HTTPFetchOngoing::complete(CURLcode res)
 	}
 
 	if (res != CURLE_OK) {
-		errorstream << "HTTPFetch for " << request.url << " failed ("
-			<< curl_easy_strerror(res) << ")" << std::endl;
+		errorstream << "HTTPFetch for " << request.url << " failed: "
+			<< curl_easy_strerror(res);
+		if (result.timeout)
+			errorstream << " (" << request.timeout << "ms)" << std::endl;
```

Comment:
so that it would still make sense if curl decided to change the wording of its error message

```suggestion
			errorstream << " (timeout = " << request.timeout << "ms)" << std::endl;
```